articles/ai-services/openai/how-to/use-web-app.md

```diff
@@ -14,27 +14,45 @@ recommendations: false
 
 # Use the Azure OpenAI web app
 
-Along with Azure OpenAI Studio, APIs, and SDKs, you can use the available standalone web app to interact with Azure OpenAI Service models by using a graphical user interface. You can deploy the app by using either Azure OpenAI Studio or a [manual deployment](https://github.com/microsoft/sample-app-aoai-chatGPT).
+Along with Azure AI Studio, Azure OpenAI Studio, APIs, and SDKs, you can use the customizable standalone web app to interact with Azure OpenAI models by using a graphical user interface. Key features include:
+* Connectivity with multiple data sources to support rich querying and retrieval-augmented generation, including Azure AI Search, Prompt Flow, and more.
+* Conversation history and user feedback collection through Cosmos DB.
+* Authentication with role-based access control via Entra ID.
+* Customization of the user interface, data sources, and features using environment variables (no-code via Azure Portal).
+* Support for modifying underlying web application source code as an open-source repository. 
+
+You can deploy the app by using either [Azure AI Studio](https://learn.microsoft.com/azure/ai-studio/tutorials/deploy-chat-web-app) or [Azure OpenAI Studio](https://learn.microsoft.com/azure/ai-services/openai/use-your-data-quickstart), or through a manual deployment through the Azure Portal or the Azure Developer CLI via your local machine [(instructions available at the repository here)](https://github.com/microsoft/sample-app-aoai-chatGPT). Depending on your deployment channel, you can pre-load a data source to chat with via the web application, but this can be changed after deployment. 
+
```

Comment:
```suggestion
You can deploy the app by using either [Azure AI Studio](/azure/ai-studio/tutorials/deploy-chat-web-app) or [Azure OpenAI Studio](/azure/ai-services/openai/use-your-data-quickstart), or through a manual deployment through the Azure Portal or the Azure Developer CLI via your local machine [(instructions available at the repository here)](https://github.com/microsoft/sample-app-aoai-chatGPT). Depending on your deployment channel, you can pre-load a data source to chat with via the web application, but this can be changed after deployment. 

```